Gemfile.lock

```diff
@@ -126,4 +126,4 @@ DEPENDENCIES
   webrick
 
 BUNDLED WITH
-   2.1.4
+   2.2.9
```

Comment:
I'm not sure what this means. Maybe @jrapala knows? I'm guessing this is fine, but I just don't know enough about Ruby to say.

Reply:
Good looking out. It's just the version of Bundler being used. We can leave this.